### ISSUE TITLE
Transpose with explicit permutation

### ISSUE THREAD

**Issue by normanrz** [manzt/zarrita.js#129]
[After some discussion](https://github.com/zarr-developers/zarr-specs/pull/264), we decided that using `C` and `F` in the `transpose` codec can be confusing when used in the codec pipeline. Therefore, we changed the spec to only allow explicit permutations in the `order` attribute. Do you think zarrita.js could also support that? Thanks!

**Comment by manzt** [comment]
Certainly! Thanks for pinging me. Has this landed in any other implementation, yet?

**Comment by normanrz** [comment]
Yep, here are a few examples:
https://github.com/scalableminds/zarrita/blob/async/zarrita/codecs.py#L370-L443
https://github.com/google/neuroglancer/tree/master/src/datasource/zarr/codec/transpose
https://github.com/clbarnes/zarr3-rs/blob/main/src/codecs/aa/transpose.rs

**Comment by manzt** [comment]
Thanks for the reference. I haven't had the chance to take a look but will soon.

**Comment by manzt** [comment]
Took me a bit, but implemented in #256 and released.
